docs/user/welcome/upgrade.rst

```diff
@@ -26,6 +26,57 @@ But first to upgrade - change your dependency to |release| (or an appropriate st
         ....
     </dependencies>
 
+GeoTools 23.x
+-------------
+
+Change to repo.osgeo.org Repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We have changed our maven repository:
+
+* Replacing *osgeo* release repository ``http://download.osgeo.org/webdav/geotools/`` with ``https://repo.osgeo.org/repository/release/``. 
+* Replacing *boundless* snapshot repository ``http://repo.boundlessgeo.com/main`` with ``https://repo.osgeo.org/repository/snapshot/``.
+
+Here is an example :file:`pom.xml`:
+
+.. code-block:: xml
+
+   <repositories>
+     <repository>
+       <id>osgeo</id>
+       <name>OSGeo Release Repository</name>
+       <url>https://repo.osgeo.org/repository/release/</url>
+       <snapshots><enabled>false</enabled></snapshots>
+       <releases><enabled>true</enabled></releases>
+     </repository>
+     <repository>
+       <id>osgeo-snapshot</id>
+       <name>OSGeo Snapshot Repository</name>
+       <url>https://repo.osgeo.org/repository/snapshot/</url>
+       <snapshots><enabled>true</enabled></snapshots>
+       <releases><enabled>false</enabled></releases>
+     </repository>
+   </repositories>
+
+Alternatively, to build older projects with no modifications to :file:`pom.xml`, configure mirrors for these repositories using :file:`~/.m2/settings.xml`:
+
+.. code-block:: xml
+
+   <mirrors>
+     <mirror>
+       <id>osgeo-release</id>
+       <name>OSGeo Repository</name>
+       <url>https://repo.osgeo.org/repository/geotools-releases/</url>
+       <mirrorOf>osgeo</mirrorOf>
+     </mirror>
+     <mirror>
+       <id>geoserver-releases</id>
+       <name>Boundless Repository</name>
+       <url>https://repo.osgeo.org/repository/Geoserver-releases/</url>
```

Comment:
This should be `geotools-snapshots`, or just `snapshot` instead of `Geoserver-releases`

Reply:
Not in this case because the content of repo.boundlessgeo.com were transferred to this specific repository `Geoserver-releases` (and yes it is an accident we cannot fix the capital G).

No `snapshots` were transferred so the above is correct.

Reply:
Okay, but the section right above describes the boundless repository as a snapshot repository, so if someone is paying attention this is likely to be a little confusing, Plus, `snapshots` will contain all the geotools snapshots, which might be used if people are only developing on a single module (which seems like a common use-case for the audience most likely to reference these docs)

Basically, this mirroring setup won't quite replicate what was there before. It might be best to just use `releases` and `snapshots` in the config to catch everything for simplicity/reliability.

Reply:
Okay I will fix...